src/main/java/uk/gov/justice/digital/hmpps/cmd/api/client/CsrClient.kt

```diff
@@ -102,8 +73,7 @@ class CsrClient(
     log.debug("User Details: finding User ${authenticationFacade.currentUsername}, Region $region")
     val csrDetails: Collection<CsrDetailDto> = csrClient
       .get()
-      .uri("/user/details?from=$from&to=$to")
-      .header("X-Region", region)
+      .uri("/user/details/$region?from=$from&to=$to")
```

Comment:
Need to consistently call csr-api with new region in url endpoints instead of the weird header setting